evaluation.py

```diff
@@ -66,19 +66,21 @@ def stacked_predict_heatmaps(self, predict_heatmaps):
         return stacked_hourglass_heatmaps
 
     #  Saves to disk stacked predicted heatmaps and stacked ground truth heatmaps and one evaluation image
-    def save_stacked_evaluation_heatmaps(self, h, X, y, stacked_predict_heatmaps_file, stacked_ground_truth_heatmaps_file, filename):
-        predict_heatmaps=self.predict_heatmaps(h, X)
+    def save_stacked_evaluation_heatmaps(self, X, y, filename):
+        predict_heatmaps=self.predict_heatmaps(X)
         stacked_predict_heatmaps=self.stacked_predict_heatmaps(predict_heatmaps)
         stacked_ground_truth_heatmaps=self.stacked_ground_truth_heatmaps(X, y)
-
-        # Save stacked images to disk
-        plt.imsave(stacked_predict_heatmaps_file, stacked_predict_heatmaps)
-        plt.imsave(stacked_ground_truth_heatmaps_file, stacked_ground_truth_heatmaps)
+        
+        # Reshape heatmaps to 3 channels, normalize channels to [0,255]
+        stacked_predict_heatmaps = cv2.cvtColor(stacked_predict_heatmaps, cv2.COLOR_GRAY2RGB)
+        stacked_predict_heatmaps = cv2.normalize(stacked_predict_heatmaps, None, alpha=0, beta=255.0, norm_type=cv2.NORM_MINMAX, dtype=cv2.CV_32F)
+        stacked_ground_truth_heatmaps = cv2.cvtColor(stacked_ground_truth_heatmaps, cv2.COLOR_BGRA2RGB)
+        stacked_ground_truth_heatmaps = cv2.normalize(stacked_ground_truth_heatmaps, None, alpha=0, beta=255.0, norm_type=cv2.NORM_MINMAX, dtype=cv2.CV_32F)
```

Comment:
Heatmaps come in one channel, cv2 needs 2. GT comes in 4 channels (including alpha), we need 3. Heatmaps come in a wide range of values [negative, positive], we need [0,255]. GT also needs to be in [0,255]